exporters/autoexport/metrics_test.go

```diff
@@ -63,3 +73,26 @@ func TestMetricExporterOTLPOverInvalidProtocol(t *testing.T) {
 	_, err := NewMetricReader(context.Background())
 	assert.Error(t, err)
 }
+
+func TestMetricExporterPrometheus(t *testing.T) {
+	defer goleak.VerifyNone(t)
+
+	t.Setenv("OTEL_METRICS_EXPORTER", "prometheus")
+	t.Setenv("OTEL_EXPORTER_PROMETHEUS_PORT", "12345")
```

Comment:
```suggestion
	t.Setenv("OTEL_EXPORTER_PROMETHEUS_PORT", "0")
```

This should make the test less flaky 😉 

AFAIK port 0 tells to find any available port.

Reply:
Done.